src/gt4py/eve/type_validation.py

```diff
@@ -358,6 +358,8 @@ def make_is_instance_of(name: str, type_: type) -> FixedTypeValidator:
         """Create an ``FixedTypeValidator`` validator for a specific type."""
 
         def _is_instance_of(value: Any, **kwargs: Any) -> None:
+            if type_ is Any:
```

Comment:
possibly tests are wrong

Reply:
The actual reason for this was actually a bug in a previous fix for dealing with the implementation of `typing.Any` as a type in modern python & typing_extensions versions.